src/components/Contacts/ContactProfileIcon.jsx

```diff
@@ -38,7 +38,7 @@ const ContactProfileIcon = ({ contact }) => {
     try {
       await getWebIdDataset(contactInfo.webId);
       setContact(contactInfo);
-    } catch {
+    } catch (error) {
```

Comment:
Unless we're using something from the error object caught by the catch block like an error message or an error status, I don't think we need to call error. Just a catch would be enough.